server/views/applications/pages/risk-to-self/acct.njk

```diff
@@ -10,43 +10,58 @@
 
   <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
 
-  {% for acct in page.accts %}
-
-    {{ govukSummaryList({
-      card: {
-        title: {
-          text: acct.referringInstitution
-        }
-      },
-      rows: [
-        {
-          key: {
-            text: "Started:"
+  {% if page.accts | length %}
+
+    {% for acct in page.accts %}
+
+      {{ govukSummaryList({
+        card: {
+          title: {
+            text: acct.referringInstitution
           },
-          value: {
-            html: acct.createdDate
-          }
+        actions: {
+          items: [
+            {
+              href: acct.removeLink,
+              text: "Remove",
```

Comment:
If you were going to make the removal request a PUT request then I'm guessing you'd pass in a form here to the `actions` property somehow?